core-spi-test/src/main/java/org/ehcache/internal/store/StoreComputeIfAbsentTest.java

```diff
@@ -33,10 +35,25 @@ public StoreComputeIfAbsentTest(StoreFactory<K, V> factory) {
     super(factory);
   }
 
+  protected Store<K, V> kvStore;
+  protected Store kvStore2;
```

Comment:
Why a raw type here? Looks like all tests having a kvStore2 always use the raw type.